{
    "type": "bundle",
    "id": "bundle--49ecb947-1c5a-46c2-864f-22c58fe632b6",
    "objects": [
        {
            "type": "software",
            "spec_version": "2.1",
            "id": "software--f2994829-189f-53f4-ab6b-fb5c7c1cea11",
            "name": "IBM Cognos Analytics 11.2.3",
            "cpe": "cpe:2.3:a:ibm:cognos_analytics:11.2.3:*:*:*:*:*:*:*",
            "languages": [
                "en_US"
            ],
            "vendor": "ibm",
            "version": "11.2.3",
            "extensions": {
                "extension-definition--6c453e0f-9895-498f-a273-2e2dda473377": {
                    "extension_type": "property-extension",
                    "nvd_cpe": {
                        "deprecated": false,
                        "cpe23Uri": "cpe:2.3:a:ibm:cognos_analytics:11.2.3:*:*:*:*:*:*:*",
                        "lastModifiedDate": "2022-11-18T17:27Z",
                        "titles": [
                            {
                                "title": "IBM Cognos Analytics 11.2.3",
                                "lang": "en_US"
                            }
                        ],
                        "refs": [
                            {
                                "ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/233571",
                                "type": "Advisory"
                            }
                        ],
                        "deprecatedBy": [],
                        "vulnerabilities": [
                            ""
                        ]
                    }
                }
            }
        }
    ]
}